### DISCUSSION TITLE
many CLOSE_WAIT

### DISCUSSION THREAD

**Discussion by tab-1** [fastapi/fastapi#7228] [Questions]
I wrote an API and started it with uvicorn. After the client was disconnected due to a large area of timeout, my FastAPI server showed a lot of CLOSE_WAIT

**Comment by Kludex** [comment]
DiCaprio, can you provide more information? If you don't know what to provide, follow our [question issue template](https://github.com/tiangolo/fastapi/blob/master/.github/ISSUE_TEMPLATE/question.md), please.

**Comment by tiangolo** [comment]
Sorry, I can't help you if you don't follow the issue template and provide more information.

If you still have problems please create a new issue and follow the template step by step.

Given that, I'm gonna close this one.
